docs/source/providers/ocp.rst

```diff
@@ -1,15 +1,16 @@
 Adding an OCP Provider
 #######################
 
-This section describes how to configure your Openshift Container Platform (OCP) cluster to provide Koku operator metering usage data.  Configuring your OCP cluster involves configuring four setup steps. Obtaining a login token for your metering-operator service account. Downloading our Ansible playbook and performing the setup step to configure connectivity to the OCP cluster and generate report resources. Create a cron job that will collect the operator metering usage data on an interval and send this data to our upload service. Create an OCP provider in Koku.
+This section describes how to configure your Openshift Container Platform (OCP) cluster to provide Koku operator metering usage data.  Configuring your OCP cluster involves configuring four setup steps. Obtaining a login token for your reporting-operator service account. Downloading our Ansible playbook and performing the setup step to configure connectivity to the OCP cluster and generate report resources. Create a cron job that will collect the operator metering usage data on an interval and send this data to our upload service. Create an OCP provider in Koku.
 
 Dependencies
 ************
 
-We require the following dependencies in order to obtain the OCP operator metering usage data. 
+We require the following dependencies in order to obtain the OCP operator metering usage data.
 
 - You must be running OCP version 3.11 or newer.
 - You must `install Operator Metering <https://github.com/operator-framework/operator-metering/blob/master/Documentation/install-metering.md>`_ on your OCP cluster.
+- The Operator Metering installation must `expose a route <https://github.com/operator-framework/operator-metering/blob/master/Documentation/configuring-reporting-operator.md#openshift-route>`_ for the reporting operator.
```

Comment:
This step is part of the operator metering install process, so I don't know if it actually belongs in the same bullet as installing Operator Metering?

Reply:
I think it can go either way -- having it a separate bullet highlights it a bit more as a prerequisite, so I vote to keep it the way you wrote it.